### PR TITLE
running py3-pyflakes on smc_sagews and fixing syntax errors

### PR DESCRIPTION
**Pull request by fchapoton**
# Description
I have fixed most of the py3 syntax errors, in a python2 compatible way

# Testing Steps
1.
1.
1.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
- [ ] No debugging console.log messages.
- [ ] All new code is actually used.
- [ ] Non-obvious code has some sort of comments.

Front end:
- [ ] Restart at least one project and check its `~/.smc/local_hub/local_hub.log`
- [ ] Completely restart Webpack with `./w` in `/src`
- [ ] Completely restart the hub by killing and restarting `./start_hub.py` in `/src/dev/project`
- [ ] Screenshots if relevant.

### CODE REVIEW

**Comment by haraldschilly** [comment]
This pull request **fixes 5 alerts** when merging 308f0d3f5586832637ac30c216135c88df4f3a83 into 077c2c6184fe28b5d325c032976b2fc8a997fc00 - [view on LGTM.com](https://lgtm.com/projects/g/sagemathinc/cocalc/rev/pr-dbaa4591dd4ba1b8bbe161aa81131b04c9b6f80e)

**fixed alerts:**

* 2 for Module is imported more than once
* 1 for Unused local variable
* 1 for Unused import
* 1 for Variable defined multiple times

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/smc_sagews/smc_sagews/julia.py [hunk 1]
```diff
@@ -164,15 +164,6 @@ def __reduce__(self):
         """
         return reduce_load_Julia, tuple([])
 
-    def _function_class(self):
```
Comment (williamstein) [inline src/smc_sagews/smc_sagews/julia.py]: **Scared** Why are you deleting this function?  What the heck?  Isn't it part of what one is supposed to implement for an interface...

Comment (fchapoton) [reply src/smc_sagews/smc_sagews/julia.py]: This was a duplicate method or function, if I remember correctly.

**Comment by haraldschilly** [comment]
hmm, I wanted to merge this with upstream's master to update it. I thought I am able to push to someone else's branch, when it is part of a PR? hmm … 

doesn't seem to work. 

at least, I haven't forgotten about this.

File: src/smc_sagews/smc_sagews/julia.py (hunk 1 above)
Comment (slel) [reply src/smc_sagews/smc_sagews/julia.py]: Indeed the method is defined twice, once at line 167 and once at line 258:
- https://github.com/sagemathinc/cocalc/blob/master/src/smc_sagews/smc_sagews/julia.py#L167
- https://github.com/sagemathinc/cocalc/blob/master/src/smc_sagews/smc_sagews/julia.py#L258

**Comment by haraldschilly** [comment]
continued in #3932

**Pull request closed without merging**